### PR TITLE
[Merged by Bors] - doc: fix many more mathlib3 names in doc comments

### PR DESCRIPTION
**Pull request by grunweg**
A mix of various changes; generated with a script and manually tweaked.

--------

Commits can be reviewed individually; sometimes, they're grouped by theme, sometimes by directory.

---

[Open in Gitpod](https://gitpod.io/from-referrer/)

### CODE REVIEW

File: Mathlib/MeasureTheory/Integral/Layercake.lean [hunk 1]
```diff
@@ -37,18 +37,21 @@ are also included.
 
 ## Main results
 
- * `lintegral_comp_eq_lintegral_meas_le_mul` and `lintegral_comp_eq_lintegral_meas_lt_mul`:
+ * `MeasureTheory.lintegral_comp_eq_lintegral_meas_le_mul`
+   and `MeasureTheory.lintegral_comp_eq_lintegral_meas_lt_mul`:
    The general layer cake formulas with Lebesgue integrals, written in terms of measures of
    sets of the forms {ω | t ≤ f(ω)} and {ω | t < f(ω)}, respectively.
- * `lintegral_eq_lintegral_meas_le` and `lintegral_eq_lintegral_meas_lt`:
+ * `MeasureTheory-lintegral_eq_lintegral_meas_le` and
```
Comment (j-loreaux) [inline Mathlib/MeasureTheory/Integral/Layercake.lean:44]:
```suggestion
 * `MeasureTheory.lintegral_eq_lintegral_meas_le` and
```

File: Mathlib/Topology/Order/IntermediateValue.lean [hunk 2]
```diff
@@ -203,7 +203,7 @@ theorem mem_range_of_exists_le_of_exists_ge [PreconnectedSpace X] {c : α} {f :
 
 In this section we prove the following results:
 
-* `IsPreconnected.ordConnected`: any preconnected set `s` in a linear order is `ord_connected`,
+* `IsPreconnected.ordConnected`: any preconnected set `s` in a linear order is `ordConnected`,
```
Comment (j-loreaux) [inline Mathlib/Topology/Order/IntermediateValue.lean:206]:
```suggestion
* `IsPreconnected.ordConnected`: any preconnected set `s` in a linear order is `OrdConnected`,
```

File: Mathlib/Topology/Order/IntermediateValue.lean [hunk 3]
```diff
@@ -335,7 +335,7 @@ theorem setOf_isPreconnected_subset_of_ordered :
 /-!
 ### Intervals are connected
 
-In this section we prove that a closed interval (hence, any `ord_connected` set) in a dense
+In this section we prove that a closed interval (hence, any `ordConnected` set) in a dense
```
Comment (j-loreaux) [inline Mathlib/Topology/Order/IntermediateValue.lean:338]:
```suggestion
In this section we prove that a closed interval (hence, any `OrdConnected` set) in a dense
```

**Review by j-loreaux — COMMENTED** [review]
Thanks

bors d+

**Comment by grunweg** [comment]
Thanks for the quick and careful review.
bors r+

**Pull request closed without merging**